sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2392,5 +2392,13 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
         }
       }
     }
+    test("load command invalid path validation ") {
```

Comment:
nit: insert an empty line before the test

Reply:
fixed. thanks